apps/src/lib/ui/SchoolInfoConfirmationDialog.jsx

```diff
@@ -73,13 +73,21 @@ class SchoolInfoConfirmationDialog extends Component {
   }
 
   closeModal = () => {
-    analyticsReporter.sendEvent(EVENTS.UPDATE_SCHOOL_INFO_DIALOG_CLOSED);
+    analyticsReporter.sendEvent(
+      EVENTS.UPDATE_SCHOOL_INFO_DIALOG_CLOSED,
+      {},
```

Comment:
Because the Platform (statsig, amplitude or both) is the third param and we are not updating the function call, we must add an empty data block param to fill the second variable:)